### DISCUSSION TITLE
How to add new administration route, such as: Intra-articular injection ?

### DISCUSSION THREAD

**Discussion by lynnieyang** [Open-Systems-Pharmacology/Forum#1528] [Questions & Problems]
Although there is the option as ''User Defined'' at the part of ''Create administration protocol'', the target organ and target compartment did not include the option for the knee. Is there possible to create administration route, such  as intra-articular injection directly to the knee compartment? 

Many thanks for the answers!

**Comment by StephanSchaller** [comment]
Currently, there is no knee compartment in PK-SIm. We had developed a synovial fluid model sometime in the past, but have yet to publish this.

**Reply by lynnieyang** [reply]
Thank you for the kindly response.

**Reply by lynnieyang** [reply]
@StephanSchaller May I ask, is there any publish planning for the synovial fluid model so far? Thank you!
